server/src/main/java/org/kaazing/gateway/server/config/parse/translate/CommunityGatewayConfigTranslatorFactorySpi.java

```diff
@@ -51,16 +54,25 @@ public GatewayConfigTranslator getTranslator(GatewayConfigNamespace ns) {
             result = new GatewayConfigTranslatorPipeline();
             GatewayConfigTranslator september2014Translator = new September2014ToNovember2015Translator();
             result.addTranslator(september2014Translator);
+            ns = GatewayConfigNamespace.NOVEMBER_2015;
```

Comment:
I would suggest switching this to a case statement as it will be cleaner and more maintainable